scripts/sql/94_update_cronjobjob_to_jobcronjob.up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE chart_ref_metadata SET "chart_name" = replace("chart_name", 'CronJob & Job', 'Job & CronJob');
+UPDATE chart_ref SET "name" = replace("name", 'CronJob & Job', 'Job & CronJob');
```

Comment:
UPDATE chart_ref SET name =  'CronJob & Job'  where name = 'Job & CronJob' and user_uploaded = false;